lookup_plugins/oo_option.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+# vim: expandtab:tabstop=4:shiftwidth=4
+
+'''
+oo_option lookup plugin for openshift-ansible
+
+Usage:
+
+    - debug:
+      msg: "{{ lookup('oo_option', '<key> default=<default_value>') }}"
```

Comment:
@twiest I'm not sure that would be sufficient, since the lookup plugin is testing multiple variables...  I think the proper solution will be to keep the default parameter and adding a boolean parameter to provide the same behavior as the default filter.

Examples:
- invalid env variable value overriding default
  - Playbook:

    ```
    name:   "{{ lookup('oo_option', 'image_name') |  default('CentOS-7-x86_64-GenericCloud.qcow2', true) }}"
    ```
  - Command invocation:

    ```
    export IMAGE_NAME=''
    bin/cluster <blah>
    ```
  - Would result in image_name being set to an empty string
- invalid -o option overriding valid env variable
  - Playbook:

    ```
    name:   "{{ lookup('oo_option', 'image_name default=CentOS-7-x86_64-GenericCloud.qcow2') }}"
    ```
  - Command invocation:

    ```
    export IMAGE_NAME=CentOS-7-x86_64-Custom.qcow2
    bin/cluster <blah> -o 'image_name=""'
    ```
  - Would result in image_name being set to an empty string